Cargo.toml

```diff
@@ -40,8 +42,8 @@ shadow-rs = "0.16.1"
 [features]
 default = ["sample_kbc"]
 sample_kbc = []
-cc_kbc = []
+cc_kbc = ["sev", "raw-cpuid"]
```

Comment:
I'm not sure whether this could cause the same issue https://github.com/confidential-containers/attestation-agent/pull/116 tries to fix in tdx. For example, when a user wants to build cc-kbc on a non-sev machine, the default feature `sev` here for `cc_kbc` might try to find sev-architecture-specific dependencies (especially c-rust calls), which might let the build fail.

Reply:
You should be able to build `sev` crate on any KVM host. Btw if we need TDX to build the TDX attester, how are we going to build it on the CI or when packaging the initrd?

Reply:
Yes, you're right. The key point I want here is that some dependencies should also be installed to help the compile process, although the machine used for CI doesn't support that kind of TEE. This seems weird at the first glance, but it is reasonable for we can use the compiled binary on different kinds of attester machines out-of-the-box-ly.